data/features.yaml

```diff
@@ -14,6 +14,10 @@
 #       tags: []
 #       create_view: False
 #       aggregate: False
+#       overwrite_store_as: True
+#       overwrite_and_merge_store_as: False
+#       store_type_list: False
+#       keep_multimatch: False
```

Comment:
can you explain a bit what these mean in the comment section (see other fields and their explanations below in the file)